pkg/gofr/cmd/request_test.go

```diff
@@ -28,17 +28,25 @@ func TestRequest_Bind(t *testing.T) {
 }
 
 func TestRequest_Param(t *testing.T) {
-	r := Request{params: map[string]string{"name": "gofr"}}
-
-	resp := r.Param("name")
+	r := NewRequest([]string{"-name=gofr", "-valid=true", "-value=12", "-test"})
+
+	testCases := []struct {
+		param string
+		want  string
+	}{
+		{"name", "gofr"},
+		{"valid", "true"},
+		{"value", "12"},
+		{"test", "true"},
```

Comment:
case can also be added about behaviour when we have keys with special characters and empty keys, same for values as well.